### PR TITLE
Change how re-joining visitors are treated on public

### PR DESCRIPTION
**Pull request by gsvr**
If a user has been registered at a Clinic at some point, but now have 0 Active Subscriptions, when they log on to the Public line, they should be treated as if not registered, but skip the Language screen.

### CODE REVIEW

File: src/personal.js [hunk 1]
```diff
@@ -331,12 +345,12 @@ go.app = function() {
                                     return self.im.contacts.save(self.contact);
                                 })
                                 .then(function() {
-                                    if (!self.im.config.faq_enabled && !self.im.config.detailed_data_collection){
+                                    if (!self.im.config.faq_enabled){
```
Comment (gsvr) [inline src/personal.js:348]: @imsickofmaps I'm quite sure this check I removed is redundant, do you agree?

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: go-app-personal.js [hunk 2]
```diff
@@ -847,7 +847,24 @@ go.utils = {
...
+            to_addr: contact.msisdn
+        };
+        return go.utils
+            .control_api_call("get", params, null, 'subscription/', im)
+            .then(function(json_result) {
+                var subs = JSON.parse(json_result.data);
+                var active = 0;
+                for (i=0;i<subs.objects.length;i++) {
+                    if (subs.objects[i].active === true) {
+                        active += 1;
```
Comment (imsickofmaps) [inline go-app-personal.js]: `active++`

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**